framework/tester.py

```diff
@@ -0,0 +1,163 @@
+import unittest
+import time
+
+from helpers import tempesta, control, stateful, tf_cfg
+
+from . import wrk_client, nginx_server
+from . import deproxy_client, deproxy_server, deproxy_manager
+from templates import fill_template
+
+__author__ = 'Tempesta Technologies, Inc.'
+__copyright__ = 'Copyright (C) 2018 Tempesta Technologies, Inc.'
+__license__ = 'GPL2'
+
+class TempestaTest(unittest.TestCase):
+    """ Test for tempesta """
+
+    backends = []
+
+    clients = []
+
+    tempesta = {
+        'listen_ip' : 'default',
+        'listen_port' : 80,
+        'backends' : [],
+    }
+
+    __servers = {}
+    __clients = {}
+    __tempesta = None
+    __deproxy_manager = deproxy_manager.DeproxyManager()
+
+    def __create_client_deproxy(self, client):
+        addr = fill_template(client['addr'])
+        port = int(fill_template(client['port']))
```

Comment:
`client['port']` in `fill_template()` should be a string, sholdn't it?

Reply:
yes, but `port` variable should be a number